schemas/BUILD

```diff
@@ -0,0 +1,5 @@
+filegroup(
+    name = "schemas",
+    srcs = glob(["*.json"]),
```

Comment:
Bazel globs can have some drawbacks, that is why it would be better to explicitly list the files.
According to the bazel style guide this specific glob type would be fine but my experience that code with globs tends to not define properly the dependencies and not being aware of what is included with a target.
https://docs.bazel.build/versions/main/skylark/build-style.html#globs